src/main/webapp/step3.html

```diff
@@ -0,0 +1,55 @@
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Home</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <nav>
+
+        </nav>
+        <div class="content">
+            <h2 class="title">Iterable Overview</h2>
+            <div class="main-inteface">
+                <div class="event-container">
```

Comment:
I believe it is mock events, are they?

Reply:
Yeah, they are mock events!

Reply:
Doesn't have to happen here, but I would make a FakeEventProvider in the javascript that provides this data from the JS rather than hardcoding it here.

Reply:
Also put a comment in the html that these are fake events with a "TODO(remusn): Replace this with a FakeEventProvider"

Reply:
The FakeEventProvider can also be used in testing rather than the real EventProvider

Reply:
I think that you can maybe use the data I used in my js files as your FakeEventProvider (Renderer class).

Reply:
Is it okay if I do this on the next PR? That PR was just for creating the HTML and CSS skeleton. On the next one, I will introduce the JS renderer class for this and there I can add the FakeEventProvider!

Reply:
Resolved!
TODO: reference the new PR as soon as it's done

Reply:
#14